build/make_bundle.sh

```diff
@@ -126,7 +126,7 @@ cat << EOF >$YAML_SCRIPT_FILE
 EOF
 
 echo Updating $OLM_CATALOG
-operator-sdk generate csv --csv-version $VERSION --operator-name splunk --update-crds --verbose
+operator-sdk generate csv --csv-version $VERSION --operator-name splunk --update-crds --make-manifests=false --verbose
```

Comment:
Does this fix this issue with `make generate`:
```
jrybczynski-MBP-3EEC4:splunk-operator$ make generate
Running operator-sdk generate k8s
[Deprecation Notice] Operator SDK has a new CLI and project layout that is aligned with Kubebuilder.
...
Command "csv" is deprecated, use 'generate bundle' or 'generate packagemanifests' instead
DEBU[0000] Debug logging is set                         
INFO[0000] Generating CSV manifest version 0.2.1        
FATA[0000] error generating CSV: error reading existing ClusterServiceVersion base deploy/olm-catalog/splunk/0.2.1/splunk.clusterserviceversion.yaml: open deploy/olm-catalog/splunk/0.2.1/splunk.clusterserviceversion.yaml: no such file or directory 
make: *** [generate] Error 1
```

Reply:
I've haven't tested those commands out yet (was considering leaving that transition for v0.19.x where we will transition to the Kubebuilder project layout) it is probably worth looking into though. 

Also interesting, I don't actually see that deprecation notice in my environment
```
splunk-operator$ make generate
Running operator-sdk generate k8s
INFO[0000] Running deepcopy code-generation for Custom Resource group versions: [enterprise:[v1], ]
INFO[0142] Code-generation complete.
Running operator-sdk generate crds
INFO[0000] Running CRD generator.
INFO[0006] CRD generation complete.
Updating deploy/olm-catalog
DEBU[0000] Debug logging is set
INFO[0000] Generating CSV manifest version 1.0.0
FATA[0202] error generating CSV: error generating ClusterServiceVersion base metadata: failed to set CRD descriptors for enterprise.splunk.com/v1beta1, Kind=ClusterMaster: no package found for v1beta1
make: *** [generate] Error 1

splunk-operator$ operator-sdk version
operator-sdk version: "v0.18.2", commit: "f059b5e17447b0bbcef50846859519340c17ffad", kubernetes version: "v1.18.2", go version: "go1.13.10 darwin/amd64"
```

Reply:
Just wondering since I was running into similar issues.  The deprecated message is probably because the `operator-sdk` version I'm using is/was 0.19.0:
```
jrybczynski-MBP-3EEC4:operator-sdk$ /usr/local/bin/operator-sdk.orig version
operator-sdk version: "v0.19.0", commit: "8e28aca60994c5cb1aec0251b85f0116cc4c9427", kubernetes version: "v1.18.2", go version: "go1.14.4 darwin/amd64"
```
Honestly I'm not sure why I was using 0.19.0 in the first place but it works fine for builds, just complains when I try to generate.  It also has issues with running `yq`.

Reply:
Ah yes, that would make sense if you are running v0.19.0 since that is the first version in which Operator SDK transitioned project formats. I don't see any issues with yq when running generate with  v0.18.2, that will be good to keep in mind though for the eventual v0.19.x migration..